exporters/etw/include/opentelemetry/exporters/etw/etw_tracer.h

```diff
@@ -214,17 +214,38 @@ class Tracer : public opentelemetry::trace::Tracer,
     {
       size_t idx = 0;
       std::string linksValue;
+
+      // reserve space for all the SpanLinks.
+      // A single SpanLink will be outptut as:
+      // [{"toSpanId":"9a43c801557f26b7","toTraceId":"ac6cd70ac4bb168a99cb7651b048d965"}]
+      // The second and above link output to string are 1 byte less than the first link.
+      const size_t linkValueSize = 80;
```

Comment:
nit -where this 80 is coming from, perhaps mention in comment? Also naming guideline - https://google.github.io/styleguide/cppguide.html#Constant_Names

Reply:
Changed the name, and the new name also documents the purpose of this variable.